pkgs/by-name/st/stown/package.nix

```
@@ -0,0 +1,46 @@
+{
+  fetchFromGitHub,
+  lib,
+  pkgs,
+  python3,
```

```suggestion
  python3Packages,
```
Usually, Python packages use `python3Packages` instead of `python3.pkgs`.